### PR TITLE
Datetime secpart

### PR DESCRIPTION
**Pull request by shlomi-noach**
If `DATETIME` includes a subsecond resolution value that is not `0`, then the returned value includes the subsecond precision & value.

### CODE REVIEW

**Comment by shlomi-noach** [comment]
closes https://github.com/siddontang/go-mysql/issues/89

**Comment by siddontang** [comment]
LGTM
PTAL @qiuyesuifeng

File: replication/row_event.go [hunk 1]
```diff
@@ -656,7 +656,7 @@ func decodeDatetime2(data []byte, dec uint16) (string, int, error) {
 	}
 
 	//ingore second part, no precision now
-	//var secPart int64 = tmp % (1 << 24)
+	var secPart int64 = tmp % (1 << 24)
```
Comment (qiuyesuifeng) [inline replication/row_event.go]: So comment line 658 can be removed now?

**Comment by qiuyesuifeng** [comment]
LGTM
@shlomi-noach Thanks.

**Pull request merged**